### ISSUE TITLE
separate schedules for thermostat devices

### ISSUE THREAD

**Issue by TERM4X** [sinricpro/feature-requests#10]
Is possible to create some schedules ONLY inside a thermostat? It would be nice to do like for the Google nest that you can decide based on the day, and on the period, wich target temperature to set, all in automatic. I've only seen the "schedule" section, but it is too much general and messy (for the whole schedules that a thermosta would need).
Is it possible or will it be possible to do it?

**Comment by kakopappa** [comment]
Hi @TERM4X  Thanks for the idea. 

I assume you are looking for something like this. We will look into adding it in the future.

![image](https://user-images.githubusercontent.com/7474406/194680964-aa062c80-ef4d-4091-8cc2-0671054e276d.png)

**Comment by TERM4X** [comment]
yes, I was exactly looking for somehting like this, thanks you for considering my idea :)
